scaffolds/javascript/package.json

```diff
@@ -6,11 +6,13 @@
   "version": "0.0.1",
   "main": "src/index.js",
   "scripts": {
-    "test": "mocha",
+    "test": "eslint src/* test/*; mocha",
     "start": "node src/index.js"
   },
   "devDependencies": {
+    "eslint": "^4.8.0",
     "mocha": "^2.5.3",
     "should": "^9.0.0"
-  }
+  },
+  "dependencies": {}
```

Comment:
Is it useful to have an empty "dependencies" section?

Reply:
It appeared by itself, I believe when I did `npm install --save-dev`. Since npm insists on it's presence, I thought I'd indulge :)